CHANGELOG/7.5.md

```diff
@@ -0,0 +1,34 @@
+# 7.5 Changelog
```

Comment:
I think we should rename the current preview.md to 7.5.md and create a new preview.md to contain only the 7.6 preview change log. In that way, the 7.5.md will have all the 7.5 preview history.